### ISSUE TITLE
[Gen 4] Searcher giving wrong results with Sync researches.

### ISSUE THREAD

**Issue by RichardPaulAstley** [Admiral-Fish/PokeFinder#232]
Got the case with an user in the RNG discord.
Case : Shiny Azelf. Gen 4. 

The user searched with Sync (the screens will be from my PokeFinder)

![image](https://user-images.githubusercontent.com/25870563/154578386-50404dee-f2ac-4aa5-b69d-5080bec6c40a.png)

The results of the researches

vs 

The result on the Generator : 

![image](https://user-images.githubusercontent.com/25870563/154578456-a93b2cd2-fa58-44fb-a2d6-c8712bf79f3b.png)

Searching with no lead gives results : 

![image](https://user-images.githubusercontent.com/25870563/154578584-614d75d4-fa93-46ff-b20f-faf445a4e8f0.png)

tldr; Searcher is broken when you search with Sync spreads. Generator gives valid result working in-game

Edit : Same issue with RNG Reporter 

![image](https://user-images.githubusercontent.com/25870563/154580370-bb6cfe4a-d062-4f51-80bf-418ae371fc33.png)

I'm gonna add more details as soon i've them but major issue.
